build/tools/certgen.sh

```diff
@@ -48,6 +48,54 @@ genCertAndKey() {
     genCert $name
 }
 
+GenSpecificCaAndCert() {
```

Comment:
where is the function called？

Reply:
+1

Reply:
What's the Specific mean?

Reply:
We can add Specific File Name as the first param when we call the function.
For example, when we need generate a set of certification and key for router's server, execute "$./certgen.sh GenSpecificCaAndCert router",we can get three files: /etc/kubeedge/ca/routerCA.crt,/etc/kubeedge/certs/router.crt,/etc/kubeedge/certs/router.key